feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.ts

```diff
@@ -21,8 +22,11 @@ export class ConfiguratorAttributeDropDownComponent
   attributeDropDownForm = new FormControl('');
   @Input() group: string;
 
-  constructor(protected quantityService: ConfiguratorAttributeQuantityService) {
-    super(quantityService);
+  constructor(
+    protected quantityService: ConfiguratorAttributeQuantityService,
+    protected translation: TranslationService
+  ) {
```

Comment:
do we have a task for constructor migration, yet?

Reply:
[#15689 Cover breaking changes due to introduction of support of additional values](https://github.com/SAP/spartacus/issues/15689)